### PR TITLE
build: upgraded angular core

### PR DESCRIPTION
**Pull request by SilverSlytherin1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage": "~2.0.3",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "ng-packagr": "^12.2.6",
+    "protractor": "~7.0.0",
+    "standard-version": "~9.1.0",
+    "ts-node": "~8.3.0",
+    "tslint": "~6.1.0",
+    "typescript": "^4.2.4"
```
Comment (llorenspujol) [inline package.json:66]: Better replace '^' for '~'. We don't want new minor version to be installed, only patches.

**Comment by llorenspujol** [comment]
Nice work! Before merging, would be nice if you rebase/squash all this 5 commits into just a single one.

**Comment by llorenspujol** [comment]
I added some small changes, like no implicit development configuration. I would suggest you rebase/squase all that 6 commits into 1, saying something like "Upgraded to Angular v12". It has to be done by you because If I do it myself you will lose the contribution authority, as happened before with another contributor. After that I would merge that commit. Thanks!

**Comment by SilverSlytherin1** [comment]
Hi! I've squashed the commits, please do a quick check before merging.
Glad to give an helping hand.

**Comment by llorenspujol** [comment]
Thanks @SilverSlytherin1 ! I added one small commit that I found about the production build in Angular v12, but nothing much important. You are now a contributor of angular-grid-layout 👏

**Pull request merged**